.github/workflows/android-build-test.yml

```diff
@@ -14,11 +14,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Set up Go
-      uses: actions/setup-go@v2
```

Comment:
will there be issues that you were using `setup-go@v2` and now in `action.yml` you're using `setup-go@v3`?

Reply:
good point we don't need to bump it